daemon/logging/build_logging.go

```diff
@@ -60,22 +61,36 @@ func OutletsFromConfig(in config.LoggingOutletEnumList) (*logger.Outlets, error)
 
 }
 
+type Subsystem string
+
 const (
-	SubsysReplication = "repl"
-	SubsysStreamrpc   = "rpc"
-	SubsyEndpoint     = "endpoint"
+	SubsysReplication   Subsystem = "repl"
+	SubsyEndpoint       Subsystem = "endpoint"
+	SubsysPruning       Subsystem = "pruning"
+	SubsysSnapshot      Subsystem = "snapshot"
+	SubsysTransport     Subsystem = "transport"
+	SubsysTransportMux  Subsystem = "transportmux"
+	SubsysRwccmd        Subsystem = "rwccmd"
+	SubsysRPC           Subsystem = "rpc"
+	SubsysControlServer Subsystem = "rpc.ctrl"
+	SubsysDataServer    Subsystem = "rpc.data"
 )
 
 func WithSubsystemLoggers(ctx context.Context, log logger.Logger) context.Context {
-	ctx = replication.WithLogger(ctx, log.WithField(SubsysField, "repl"))
-	ctx = streamrpc.ContextWithLogger(ctx, streamrpcLogAdaptor{log.WithField(SubsysField, "rpc")})
-	ctx = endpoint.WithLogger(ctx, log.WithField(SubsysField, "endpoint"))
-	ctx = pruner.WithLogger(ctx, log.WithField(SubsysField, "pruning"))
-	ctx = snapper.WithLogger(ctx, log.WithField(SubsysField, "snapshot"))
-	ctx = serve.WithLogger(ctx, log.WithField(SubsysField, "serve"))
+	ctx = replication.WithLogger(ctx, log.WithField(SubsysField, SubsysReplication))
+	ctx = endpoint.WithLogger(ctx, log.WithField(SubsysField, SubsyEndpoint))
+	ctx = pruner.WithLogger(ctx, log.WithField(SubsysField, SubsysPruning))
+	ctx = snapper.WithLogger(ctx, log.WithField(SubsysField, SubsysSnapshot))
+	ctx = transport.WithLogger(ctx, log.WithField(SubsysField, SubsysTransport))
+	ctx = transportmux.WithLogger(ctx, log.WithField(SubsysField, SubsysTransportMux))
+	ctx = rwccmd.ContextWithLog(ctx, log.WithField(SubsysField, SubsysRwccmd))
```

Comment:
rwccmd uses Printf which results in error-level log output. we'll want to have an adaptor or something to map this to debug-level output.